src/App.js

```diff
@@ -4,7 +4,7 @@ import "styles/app.scss";
 
 function App() {
   return (
-    <div>
+    <div className="min-h-screen bg-violet3">
```

Comment:
Bhai, does h stands for height? Can we have a bit more specific naming here?
```suggestion
    <div className="min-height-screen bg-violet-3">
```

Reply:
Tried to maintain css classname consistent by following [tailwind css](https://tailwindcss.com/docs/min-height) `classname` naming convention. However for better understanding changed the name according to the suggestion. @fza-wtag vai

Reply:
Got it. Please feel free to go for height as h. Just wanted to make sure that I have understood it properly or not. Thanks for sharing.